### PR TITLE
[15 minute fix] Reading list: fix broken pagination

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Reading list pagination is currently broken, see https://github.com/forem/forem/issues/14016

## Related Tickets & Documents

Closes https://github.com/forem/forem/issues/14016

## QA Instructions, Screenshots, Recordings

1. You need to add at least 80 articles to your reading list
2. Click "Load More"
3. See that the new page is loaded

https://user-images.githubusercontent.com/146201/122744874-ba6ca680-d288-11eb-944e-fe9319464971.mov

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: we don't really have E2E tests for the reading list and it's currently a low priority
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

**Review by Ridhwana — APPROVED** [review]
@rhymes this looks good, is there no test that we can update to ensure that we don't regress at any point?

**Comment by rhymes** [comment]
@Ridhwana not currently :(

**Review by jacobherrington — APPROVED** [review]
Nice catch! :+1:

File: app/javascript/searchableItemList/searchableItemList.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ export function loadNextPage() {
   component.setState({ page: page + 1 });
   component.search(query, {
     page: page + 1,
-    tags: [selectedTag],
+    tags: selectedTag ? [selectedTag] : [],
```
Comment (citizen428) [inline app/javascript/searchableItemList/searchableItemList.js:133]: **suggestion (non-blocking):** If we're only guarding against `selectedTag` being `nil` we could do that without the ternary:

```suggestion
    tags: [selectedTag].compact,
```

**Review by citizen428 — APPROVED** [review]

File: app/javascript/searchableItemList/searchableItemList.js (hunk 1 above)
Comment (rhymes) [reply app/javascript/searchableItemList/searchableItemList.js:133]: I wouldn't find the `.compact` method in JS :)

Comment (citizen428) [reply app/javascript/searchableItemList/searchableItemList.js:133]: OMG, I was still a bit sleepy when I wrote this, this isn't actually Ruby lol

**Pull request merged**